canonical-websites/jaas.ai
# Update Azure logo

## Done

- Update the Azure logo on the home page

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- In the Deploy section, the group of logos should contain the updated Azure logo (the one with the symbol and "Azure" not "Microsoft Azure" (see screenshot).

![deploy-logos-2x](https://user-images.githubusercontent.com/361637/55763758-bee38500-5aab-11e9-8651-29cb44e881ff.png)

## Details

- Fixes: #486.
